libxrdp/libxrdp.c

```diff
@@ -440,10 +461,26 @@ libxrdp_send_bitmap(struct xrdp_session *session, int width, int height,
                 }
 
                 p = s->p;
-                lines_sending = xrdp_bitmap_compress(data, width, height,
-                                                     s, bpp,
-                                                     4096 - total_bufsize,
-                                                     i - 1, temp_s, e);
+
+                if (bpp > 24)
+                {
+                    LLOGLN(10, ("libxrdp_send_bitmap: 32 bpp"));
```

Comment:
Never miss a change to print the actual value (bpp), not the assumed value (32).